release-tools/SIDECAR_RELEASE_PROCESS.md

```diff
@@ -84,7 +84,7 @@ naming convention `<hostpath-deployment-version>-on-<kubernetes-version>`.
     1. Compare the generated output to the new commits for the release to check if
        any notable change missed a release note.
     1. Reword release notes as needed, ideally in the original PRs so that the
-       release notes can be regnerated. Make sure to check notes for breaking
+       release notes can be regenerated. Make sure to check notes for breaking
```

Comment:
We can't change it here.  These need to be fixed in release-tools first and imported here.

Reply:
it's already fixed in release-tools, check here: https://github.com/kubernetes-csi/csi-release-tools/pull/255, this PR just updates the vendored release-tools